avocado/plugins/spawners/lxc.py

```diff
@@ -0,0 +1,200 @@
+import asyncio
+import os
+import shutil
+import stat
+import tempfile
+
+try:
+    import lxc
```

Comment:
Unless I'm missing something, IIUC, lcx is necessary for your code. Without it, we can't run it. So following the same logic as others imports, I would recommend to remove this try..except here and add lxc into the plugin's `requirements.txt` with instructions to install on the plugin's README.

Reply:
Hi Beraldo and thank for you for taking the time to validate the implementation! Now about this:

This is a good point but don't you think LXC should be an optional requirement for people that won't use the LXC spawners? For somebody that might want a basic Avocado install and just using process spawners it might be preferable to only have a lightweight Avocado install without building or downloading Podman/LXC dependencies. So this was an approach to keep things optional. What do you think about this idea overall? Of course, it is easier to just important the module and require the pip dependency.

Reply:
I looked around but didn't see an easy way to specify dependencies for the LXC spawner as it is just a module and not a plugin with its own `setup.py`, RPM subpackage, and so on. Currently there is only `process` and `podman` listed under the `spawners` folder, I thought it is more reasonable to simply also provide `lxc` there. Or is `podman` optional in some sense and having its own plugin dependencies specified somewhere?

Reply:
Pretty much all other plugins that have Python deps other than setuptools, will go into `optional_plugins`.  The podman spawner is a different beast because it requires a system binary (to put it simply).  The LXC plugin is a bit different because it requires the *Python* bindings, but not *only that*.  Until we have a better way of describing dependencies such as system binaries, this is OK IMO.